packages/plugin-pnp/sources/index.ts

```diff
@@ -27,6 +27,7 @@ export const getPnpPath = (project: Project) => {
   return {
     main: ppath.join(project.cwd, mainFilename as Filename),
     other: ppath.join(project.cwd, otherFilename as Filename),
+    esmLoader: ppath.join(project.cwd, `experimental-pnp-esm-loader.js` as Filename),
```

Comment:
I know that this PR is still an early draft, but is there a reason not to put this in the `.yarn` folder?

Reply:
Nope, it can be anywhere and I have no plans of keeping it at the project root